doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1404,7 +1367,12 @@ constructions.
 Definitions of recursive objects in co-inductive types
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: CoFixpoint @ident {? @binders } {? : @type } := @term
+.. cmd:: CoFixpoint @cofix_definition {* with @cofix_definition }
+
+   .. insertprodn cofix_definition cofix_definition
+
+   .. prodn::
+      cofix_definition ::= @ident_decl {* @binder } {? : @term } {? := @term } {? @decl_notations }
```

Comment:
Same remarks on `CoFixpoint`.

Reply:
I removed the cmdv.  I reworded but didn't move the statement after it.